src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -39,6 +41,13 @@ export class ResponsiveTabs implements IResponsiveComponent {
     this.bindNukeEvents();
   }
 
+  private get initialTabsOrder(): HTMLElement[] {
```

Comment:
Yeah 🤔 

I was doing this because I wanted to do this only when the components were actually initialized, but it's not really needed. I'll change it to what you are suggesting.